### PR TITLE
Improvement/dm 3652 added ignoring source port property y2024

### PR DESCRIPTION
**Pull request by shaikhahme**

### CODE REVIEW

File: content/change-logs/device-management/lwm2m-agent-10-18-0-445-added-ignoring-source-port-property.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+date:
+title: New Lwm2m Agent property to ignore packets from certain source ports
```
Comment (BeateRixen) [inline content/change-logs/device-management/lwm2m-agent-10-18-0-445-added-ignoring-source-port-property.md:3]:
```suggestion
title: New LWM2M agent property to ignore packets from certain source ports
```

File: content/change-logs/device-management/lwm2m-agent-10-18-0-445-added-ignoring-source-port-property.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3652
+version: 10.18.0.445
+---
+
+The Lwm2m Agent can now be configured to ignore packets from specific source ports. By default, several popular UDP protocols ports are blocked.
```
Comment (BeateRixen) [inline content/change-logs/device-management/lwm2m-agent-10-18-0-445-added-ignoring-source-port-property.md:18]:
```suggestion
The LWM2M agent can now be configured to ignore packets from specific source ports. By default, several popular UDP protocol ports are blocked.
```

**Review by BeateRixen — APPROVED** [review]

**Review by aaronraab — APPROVED** [review]

File: content/change-logs/device-management/lwm2m-agent-10-18-0-445-Security-improvement-ignores-predefined-source-ports.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3652
+version: 10.18.0.445
+---
+
+Security improvement for the LWM2M Agent to ignore packets from predefined source ports.
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/lwm2m-agent-10-18-0-445-Security-improvement-ignores-predefined-source-ports.md:18]:
```suggestion
To improve security, the LWM2M agent can now ignore packets from predefined source ports.
```
based on https://github.com/SoftwareAG/c8y-docs/pull/2255/files

File: content/change-logs/device-management/lwm2m-agent-10-18-0-445-Security-improvement-ignores-predefined-source-ports.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+date:
+title: Security improvement to ignore packets from certain source ports
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/lwm2m-agent-10-18-0-445-Security-improvement-ignores-predefined-source-ports.md:3]:
```suggestion
title: LWM2M agent security improvement to ignore packets from certain source ports
```

File: content/change-logs/device-management/lwm2m-agent-10-18-0-445-Security-improvement-ignores-predefined-source-ports.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    label: Improvement
+product_area: Device Management & Connectivity
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3652
+version: 10.18.0.445
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/lwm2m-agent-10-18-0-445-Security-improvement-ignores-predefined-source-ports.md:15]:
```suggestion
version: 10.18.540.156
```
according to https://github.softwareag.com/IOTA/cumulocity-agents/commits/release/y2024 somehow automation did not add to the task.
10.18.0.445 is 1018 release.

**Pull request merged**